script.musicfinder/default.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+#    This script is a modification of service.skin.widgets and service.library.data.provider
+#    Thanks to the original authors: BigNoid, Unfledged and Martijn.
+
+import os
+import sys
+import xbmc
+import xbmcgui
+import xbmcaddon
+
+try:
+    import simplejson as json
+except Exception:
+    import json
+
+__addon__        = xbmcaddon.Addon()
```

Comment:
we're also trying to get rid of the `__name__` formatting.

please use `addon = ` or `ADDON = ` in the next update